cli/admin/call.go

```diff
@@ -189,25 +192,18 @@ func printCallRes(callResRaw interface{}) {
 	}
 }
 
-func printMessage(receivedString string) {
-	parts := strings.SplitN(receivedString, "\n", 2)
-	msgEncoded := parts[1]
+func printMessage(pushedData interface{}) {
+	msg, ok := pushedData.(string)
+	if !ok {
+		log.Warnf("Intermediate message should be a string, got %v", pushedData)
 
-	var msg string
-	if err := yaml.UnmarshalStrict([]byte(msgEncoded), &msg); err != nil {
+		msgEncoded, err := yaml.Marshal(pushedData)
+		if err != nil {
+			log.Errorf("Failed to encode received intermediate message: %s", err)
+		}
```

Comment:
Pushed data is received unmarshalled, this is `interface{}` data (In case of plain text connection it is received in yaml format and then unmarshalled, for binary protocol it's received as a msgpack). But here we marshal this data to YAML to display it for user. Of course, `{a = 'b'}` table is handled properly.